camelConnector/src/test/java/io/vantiq/extsrc/camelconn/connector/VantiqConfigurationTest.java

```diff
@@ -358,6 +402,123 @@ public void testComponentInitConfigurationWithPropertyValues() {
                           });
     }
     
+    @Test
+    public void testHdrDupConfiguration() {
```

Comment:
Test that runs from configuration to operation of system (albeit in unit test mode) for header duplication.